### DISCUSSION TITLE
Let's make the first Feature request !

### DISCUSSION THREAD

**Discussion by Pverte** [Hyperchroma/hyperchroma#5] [Feature Requests]
Hi ! Just discovered your app not so long ago, and it's very good ! During my utilization, I found some ideas of features : 

 - **Be able to automatically put the cover image of the music as a background**
 - **Put Hyperchroma on a screen, and separate the menu on another window**, it's difficult to test options if you can't see the final render, blurred because of the menu

**Comment by aeroheim** [comment]
Hey, thanks for the feedback!

> Be able to automatically put the cover image of the music as a background

Added to the backlog. It will most likely be implemented as a configurable option for the background queue.

> Put Hyperchroma on a screen, and separate the menu on another window, it's difficult to test options if you can't see the final render, blurred because of the menu

I recommend toggling the menu using its hotkey when editing effects to test your changes. This is the intended workflow, and the state of the UI is always saved to facilitate it.
